db/schema.rb

```diff
@@ -1,100 +1,99 @@
-    t.integer "purpose_id",  limit: 5,  null: false
-    t.integer "transfer_id", limit: 5, null: false
+  create_table "entities", force: :cascade do |t|
+    t.string "name", limit: 255, default: "", null: false
+    t.string "legal_name", limit: 255, default: "", null: false
+    t.text "legal_other"
+    t.text "address"
+    t.text "telephone"
+    t.text "fax"
+    t.text "email"
+    t.text "dp_reg"
+    t.text "dp_contact"
+    t.text "signatory"
+    t.integer "is_eea", limit: 2, default: 0, null: false
+    t.text "country", null: false
+    t.text "area"
+    t.integer "is_areaoffice", limit: 2, default: 0, null: false
+    t.text "description"
+    t.index ["name"], name: "entityname_entities", unique: true
   end
 
-  add_index "purpose_transfers", ["purpose_id"], name: "fk_purposes_transfers_purposes_1", using: :btree
-  add_index "purpose_transfers", ["transfer_id"], name: "fk_purposes_transfers_transfers_1", using: :btree
-
-  create_table "purposes", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 ROW_FORMAT=COMPACT" do |t|
-    t.string "name",                           default: "", null: false
-    t.text   "full_description", limit: 65535,              null: false
-    t.text   "data_subjects",    limit: 65535,              null: false
-    t.text   "data_categories",  limit: 65535,              null: false
-    t.text   "data_recipients",  limit: 65535,              null: false
-    t.text   "additional_info",  limit: 65535
-    t.text   "storage_limit",    limit: 65535,              null: false
-    t.text   "data_sensitive",   limit: 65535,              null: false
-    t.text   "security_policy",  limit: 65535,              null: false
+  create_table "entity_transfers", force: :cascade do |t|
+    t.integer "entity_id", null: false
+    t.integer "transfer_id", null: false
+    t.integer "is_recipient", limit: 2, default: 0, null: false
   end
 
-  add_index "purposes", ["name"], name: "purposename", unique: true, using: :btree
-
-  create_table "transfer_types", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 ROW_FORMAT=COMPACT" do |t|
-    t.string "name",          default: "", null: false
-    t.string "template_name"
+  create_table "purpose_transfers", force: :cascade do |t|
+    t.integer "purpose_id", null: false
+    t.integer "transfer_id", null: false
   end
 
-  add_index "transfer_types", ["name"], name: "transfertypename", unique: true, using: :btree
+  create_table "purposes", force: :cascade do |t|
+    t.string "name", limit: 255, default: "", null: false
+    t.text "full_description", null: false
+    t.text "data_subjects", null: false
+    t.text "data_categories", null: false
+    t.text "data_recipients", null: false
+    t.text "additional_info"
+    t.text "storage_limit", null: false
+    t.text "data_sensitive", null: false
+    t.text "security_policy", null: false
+    t.text "context"
```

Comment:
Where did that come from? I don't see it in the source. Same question for `transfer_types.context`

Reply:
Good question, but they existed in MySQL.